### DISCUSSION TITLE
Add/Remove Git Repositories for automatic updates on extensions

### DISCUSSION THREAD

**Discussion by trymeouteh** [DevToys-app/DevToys#1291] [Ideas]
Please add the ability to add/remove any Github, Gitlab, Gitea and Forgejo repository in DevToys. By adding a repository, you can install extensions right from the git repo and have DevToys automatically update the extensions when there is a new release.

Similar to how Kodi Repositories work but for DevToys!

**Comment by veler** [comment]
Why not simply checking on updates available on the Nuget package? I was planning on implementing that next week or the following one.

**Reply by trymeouteh** [reply]
Using git repos is more decentralised than using Nuget centralised repository.

Not against having automatic updates on Nuget packages, but to be able to get a package easily from any git repo and being able to get updates from it would be a great feature
